### PR TITLE
Tip button and some JWT fixes

### LINKED ISSUE #21: Tip button
Just a though - we can probably implement some sort of `clap` button.

https://github.com/aeternity/jitsi-meet/issues/21#issuecomment-638764765

### LINKED ISSUE #79: Make chain name and display name always the same
Regarding to the https://github.com/aeternity/protocol/blob/epoch-v1.2.0/AENS.md

The chain name can be any maximum length. So we need to remove all name processing, for example that we have at participant name that use `MAX_DISPLAY_NAME_LENGTH` constant to cut it.

### PR DESCRIPTION
**Pull request by shapkarin**
closes #21 and fix #79

### CODE REVIEW

File: react/features/toolbox/components/web/Toolbox.js [hunk 1]
```diff
@@ -1394,6 +1417,7 @@ function _mapStateToProps(state) {
             || sharedVideoStatus === 'pause',
         _visible: isToolboxVisible(state),
         _visibleButtons: equals(visibleButtons, buttons) ? visibleButtons : buttons,
+        _dominantSpeakerName: getDominantSpeaker(state)?.name || '',
```
Comment (shapkarin) [inline react/features/toolbox/components/web/Toolbox.js]:
```suggestion
        _dominantSpeakerName: getDominantSpeaker(state)?.name ?? '',
```

**Comment by shapkarin** [comment]
@Chima1707 please check [this commit](https://github.com/aeternity/jitsi-meet/pull/35/commits/7bb272b65d46b466b3058b1aea12eb5dc2170a81). I hope it's cool one.

File: conference.js [hunk 2]
```diff
@@ -2185,7 +2184,7 @@ export default {
             JitsiConferenceEvents.DISPLAY_NAME_CHANGED,
             (id, displayName) => {
                 const formattedDisplayName
-                    = getNormalizedDisplayName(displayName);
```
Comment (venimus) [inline conference.js]: why did you remove it?

**Comment by venimus** [comment]
please revert the removal of `getNormalizedDisplayName` it is out of scope and not really needed
there is a constant of 54 max lenght for names which covers both .chain and ak_ addresses

Also not sure what is the feature, currently it shows a button that tips the "room" but nobody can claim it later, so for me this does not make much useful, at least i don't see how we can tip the dominant speaker

**Comment by shapkarin** [comment]
@venimus thanks. looks like the max length is around 32.

**Comment by shapkarin** [comment]
> Also not sure what is the feature, currently it shows a button that tips the "room" but nobody can claim it later, so for me this does not make much useful, at least i don't see how we can tip the dominant speaker

@venimus it use the same approach that `superhero-button` has.

So i get that error 
```
This means that the owner of the URL has not created a Superhero wallet yet. They can claim their tips as soon as they make one and add their wallet address to their website.
```

This related to my question – is some way to tip directly to the user without url. Looks like at superhero profile it's possible, I will have a look.

**Comment by shapkarin** [comment]
@venimus @mradkov @DeeJayElly this button needs to be implemented like we have at https://github.com/aeternity/superhero-ui/blob/develop/src/components/TipInput.vue but not like at https://github.com/aeternity/superhero-button

**Comment by shapkarin** [comment]
not sure where is that commit https://github.com/aeternity/jitsi-meet/commit/79bf11f49b276615fb4167ad009288d1a578c4ea but it's in the branch

**Comment by shapkarin** [comment]
changed the PR base branch helped.

File: react/features/conference/components/web/TipButton.js [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+        let url = '';
+
+        if (this.comment) {
+            url = `${URLS.SUPER}/tip/${this.comment.tipId}/comment/${this.comment.id}`;
+        } else if (this.userAddress) {
+            url = `${URLS.SUPER}/user-profile/${this.userAddress}`;
+        } else {
+            url = `${URLS.SUPER}/tip/${this.tip.id}`;
+        }
```
Comment (shapkarin) [inline react/features/conference/components/web/TipButton.js]:
```suggestion
        url = `${URLS.SUPER}/user-profile/${this.props.account}`;
```

**Comment by shapkarin** [comment]
TODO: squash

**Comment by shapkarin** [comment]
TODO: remove `_onSendTipComment`
